keps/sig-auth/1687-hierarchical-namespaces-subproject/README.md

```diff
@@ -0,0 +1,191 @@
+of this file) is [here](/keps/NNNN-kep-template/README.md).
+
+**Note:** Any PRs to move a KEP to `implementable` or significant changes once
+it is marked `implementable` must be approved by each of the KEP approvers.
+If any of those approvers is no longer appropriate than changes to that list
+should be approved by the remaining approvers and/or the owning SIG (or
+SIG Architecture for cross cutting KEPs).
+-->
+# KEP-1687: Hierarchical Namespaces As A Subproject
+
+<!--
+This is the title of your KEP.  Keep it short, simple, and descriptive.  A good
+title can help communicate what the KEP is and should be considered as part of
+any review.
+-->
+
+<!--
+A table of contents is helpful for quickly jumping to sections of a KEP and for
+highlighting any additional information provided beyond the standard KEP
+template.
+
+Ensure the TOC is wrapped with
+  <code>&lt;!-- toc --&rt;&lt;!-- /toc --&rt;</code>
+tags, and then generate with `hack/update-toc.sh`.
+-->
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+- [Goals](#goals)
+- [Proposal](#proposal)
+- [Risks and Mitigations](#risks-and-mitigations)
+- [Graduation Criteria](#graduation-criteria)
+- [Alternatives](#alternatives)
+  - [Absorb Hierarchical Namespace Controller Into Another Project](#absorb-hierarchical-namespace-controller-into-another-project)
+  - [Hierarchical Namespace Controller Continues As A Non-Sig Project](#hierarchical-namespace-controller-continues-as-a-non-sig-project)
+  - [Abandon Hierarchical Namespace Controller](#abandon-hierarchical-namespace-controller)
+- [Implementation History](#implementation-history)
+<!-- /toc -->
+
+
+## Summary
+
+[Hierarchical Namespace Controller](https://github.com/kubernetes-sigs/multi-tenancy/tree/master/incubator/hnc)
+makes it easier for you to create and manage namespaces in your cluster.
+For example, you can create a hierarchical namespace under your team's namespace,
+even if you don't have cluster-level permission to create namespaces, and easily
+apply policies like RBAC and Network Policies across all namespaces in your
+team (e.g. a set of related microservices).
+
+You can read more about hierarchical namespaces in the
+[HNC Concepts doc](https://docs.google.com/document/d/1R4rwTweYBWYDTC9UC-qThaMkPk8gtflr_ycHOOqzBQE/edit).
+
+
+## Motivation
+
+The Hierarchical Namespace Controller is currently being developed within the
+Multi-Tenancy Working Group, (of which Sig-Auth the sponsor). As Working Groups
+are not meant to own code, and Hierarchical Namespace Controller is nearly to
+MVP status, a permant home is required. Additionally, having a permanent home
+for Hierarchical Namespace Controller prior to officially releasing will prevent
+cumbersome migrations of client libraries if a move were to happen at a later time.
+
+
+## Goals
+
+Establish a new repository and permanent home for Hierarchical Namespace
+Controller at github.com/kubernetes-sigs/hierarchical-namespace-controller to be
```

Comment:
```suggestion
Controller at github.com/kubernetes-sigs/hierarchical-namespace-controller-prototype to be
```

What level of support are we proposing here?

Reply:
This is an early project (e.g. alpha), there's no commitment to bringing it to beta or GA.

I'd rather not put "prototype" in the name, otherwise if this _does_ graduate, everyone needs to change their links (again).

Reply:
I agree with @adrianludwin, our main driver here is to get a permanent home so we can publish client libraries without requiring a massive migration if we graduate.

Reply:
Let's keep the prototype out of the name.

Reply:
👍